modules/platforms/cpp/ignite/odbc/query/data_query.h

```diff
@@ -129,7 +129,27 @@ class data_query : public query
      *
      * @return true, if all cursors closed remotely.
      */
-    [[nodiscard]] bool is_closed_remotely() const;
+    [[nodiscard]] bool is_closed_remotely() const {
+        return !m_has_more_pages;
```

Comment:
In the presence of `has_more_pages()` method this method seems a bit odd.

Reply:
Agree. This is ported code so yeah, may look a bit weird. Fixed.